## Results

### Field-specific Citation Networks and Importances

We built citation networks using MeSH headings to denote fields.
We created a combined network and 100 shuffled versions for each pair of fields using a degree-preserving shuffling approach (see Methods).
We then split the original and shuffled networks to maintain the same set of papers within each constituent field, calculating the PageRank for all networks.
This process allows us to compare the manuscript's observed PageRank in each field with that constructed from a null that assumes no meaningful differences in citation patterns between fields.
Finally, we calculated percentile scores by comparing the PageRanks for the original and shuffled networks and made our results available at (TODO web server URL).

![ 
Schematic of the network analysis workflow. 
We construct networks of citations from pairs of MeSH headings, create shuffled versions of the graphs, split the networks into their component fields, and calculate the PageRank for each article in the networks.
](./images/updated_schematic.png "Workflow schematic"){#fig:workflow width="100%"}

### Journal Rankings Differ Between Fields

In an attempt to quantify the relative importance of journals, scientists have created rankings using metrics the Journal Impact Factor, which is essentially based on citations per article, and those that rely on more complex representations like Eigenfactor [@doi:10.5860/crln.68.5.7804]. 
It has previously been reported that journal rankings differ substantially between fields using metrics based on citation numbers [@doi:10.1007/s11192-022-04402-w].
We calculated a PageRank-based score for the journal as the median PageRank of manuscripts published in that journal for that field.
We first sought to understand the extent to which journal ranking differences replicated using PageRank.

To do so, we ranked the journals in each of the 45 fields in our dataset to determine whether the top ranking journal would be consistent across fields or different (Fig. @fig:jrank).
We found that the most common top-ranked journal was *Science*.
This was unsurprising, given that it tends to rank highly among global journal level metrics such as eigenfactor.
However, the ranking was very field-dependent, with only 20% of fields having *Science* as their top ranked journal.

One could argue that while general journals may have differing influence by field, specialty journals correspond to a single field so field-aware metrics are irrelevant.
That turns out to be untrue.
Of the 5,178 journals with at least 50 articles present in our dataset, only 56 (1.1%) have more than 90 percent of their articles within a single field.
One driver of the rarity in truly field-specific journals is the overlap between fields.
For example, proteomics and metabolomics are different fields with different focuses, but their techniques are often applied in conjunction with each other.
This overlap leads to papers like "The metabonomic signature of celiac disease," a paper with an extremely high PageRank in the field of metabolomics, that was published in the *Journal of Proteome Research* [@doi:10.1021/pr800548z].

![
The number of times each journal had the highest median PageRank for a field.
](./images/journal_hist.png "Differences between journals ranks"){#fig:jrank width="100%"}

### Manuscript PageRanks Differ Between Fields

We examined the univariate distributions of PageRank across fields (Fig. @fig:fieldhist).
To control for differences in the papers present, we also performed the analyses for field pairs using only the intersecting papers (Fig. @fig:pairhist).
We found that fields differed in their PageRank distributions.
We also examined the correspondence of individual papers within the field-pair intersections.
We selected field pairs with varying degrees of correlation between their PageRanks (Fig. @fig:heatmaps).
We found that many papers exhibited some correspondence between fields.
However, given the differences in univariate distributions, it remained difficult to determine which differences might be meaningful.

![
The distribution of article PageRanks in various fields
](./images/combined_heatmap.png "Article PageRanks"){#fig:fieldhist width="100%"}

![
The distribution of PageRanks for articles shared between fields.
](./images/combined_histogram.png "Article Pair PageRanks"){#fig:pairhist width="100%"}

![
Heatmap of shared papers across fields
](./images/combined_heatmap.png "Heatmaps"){#fig:heatmaps width="100%"}

We devised a strategy to generate an empirical null for a field pair under the assumption that the field pair represented a single, homogenous field.
For each field-pair intersection, we performed a degree-distribution preserving permutation.
We created 100 permuted networks for each field pair.
We then calculated a percentile using the number of permuted networks with a lower PageRank for a manuscript than the true PageRank.
A manuscript with a PageRank higher than all networks has a percentile of 100, and one lower than all permuted networks has a percentile of zero. 
<!--at this point I am very confused about how what you are describing here is different than the first subsection-->
We used the difference in the percentile in each field as the field-specific affinity for a given paper.

By examining the fields' PageRank percentiles, we found that many articles had large differences in their perception between fields.
In nanotechnology and microscopy, papers with high nanotechnology percentiles and low microscopy percentiles tended towards applications of nanotechnology, while their counterparts with high microscopy percentiles and low nanotechnology percentiles were often papers about technological developments in microscopy (Fig. @fig:percentiles A, Table 1).
Immunochemistry-favored papers are largely applications of immunochemical methods, while anatomy-favored articles tend to focus experiments on a single anatomical region (Fig. @fig:percentiles B, Table 2).
Proteomics and metabolomics tend to use similar methods, so the fields on either end are largely (though not entirely) field-specific applications of those methods (Fig. @fig:percentiles C, Table 3).
Computational biology is similarly applications-focused, though human genetics tends towards policy papers due to its MeSH heading (H01.158.273.343.385) excluding fields like genomics, population genetics, and microbial genetics (Fig. @fig:percentiles D, Table 4).
In addition to papers with large differences between fields, each field also has papers with high PageRanks and similar percentiles in both fields.
Overall it is clear that while some papers may be influential in multiple fields, others have more field-specific import.

| Nanotechnology Percentile | Microscopy Percentile | Title | Reference |
|----------|----------|----------|----------|
| 100 | 4  | A robust DNA mechanical device controlled by hybridization topology | [@doi:10.1038/415062a] |
| 100 | 5  | Bioadhesive poly(methyl methacrylate) microdevices for controlled drug delivery | [@doi:10/c7fpg4] |
| 99  | 2  | DNA-templated self-assembly of protein arrays and highly conductive nanowrires | [@doi:10.1126/science.1089389] |
| 0   | 100| Photostable luminescent nanoparticles as biological label for cell recognition of system lupus erythematosus patients | [@doi:10.1166/jnn.2002.105] |
| 5   | 90 | WSXM: a software for scanning probe microscopy and a tool for nanotechnology | [@horcas2007wsxm] |
| 0   | 77 | Measuring Distances in Supported Bilayers by Fluorescence Interference-Contrast Microscopy: Polymer Supports and SNARE Proteins | [@doi:10/dqsg2c] |
| 100 | 99 | Toward fluorescence nanoscopy | [@doi:10.1038/nbt895] |
| 100 | 86 | In vivo imaging of quantum dots encapsulated in phospholipid micelles | [@doi:10.1126/science.1077194]|
| 100 | 99 | Water-Soluble Quantum Dots for Multiphoton Fluorescence Imaging in Vivo | [@doi:10.1126/science.1083780]|

Table 1: Nanotechnology/microscopy papers of interest 

| Immunochemistry Percentile| Anatomy Percentile | Title | Reference |
|----------|----------|----------|----------|
| 100 | 45 | Immunoelectron microscopic exploration of the Golgi complex | [@doi:10.1177/31.8.6863900] |
| 100 | 14 | Immunocytochemical and electrophoretic analyses of changes in myosin gene expression in cat posterior temporalis muscle during postnatal development | [@doi:10.1007/bf01682147] |
| 98 | 5 | Electron microscopic demonstration of calcitonin in human medullary carcinoma of thyroid by the immuno gold staining method | [@doi:10.1007/bf00514331] |
| 12 | 100 | Grafting genetically modified cells into the rat brain: characteristics of E. coli Î²-galactosidase as a reporter gene | [@doi:10/dptnbm] |
| 12 | 100 | Vitamin-D-dependent calcium-binding-protein and parvalbumin occur in bones and teeth | [@doi:10.1007/bf02405306] |
| 3 | 100 | Mapping of brain areas containing RNA homologous to cDNAs encoding the alpha and beta subunits of the rat GABAA gamma-aminobutyrate receptor | [@doi:10.1073/pnas.85.20.7815] |
| 100 | 100 | Studies of the HER-2/neu Proto-Oncogene in Human Breast and Ovarian Cancer| [@doi:10.1126/science.2470152] |
| 100 | 100 | Expression of c-fos Protein in Brain: Metabolic Mapping at the Cellular Level| [@doi:10.1126/science.3131879] |
| 100 | 100 | Proliferating cell nuclear antigen (PCNA) immunolocalization in paraffin sections: An index of cell proliferation with evidence of deregulated expression in some neoplasms| [@doi:10.1126/science.3131879] |

Table 2: Immunochemistry/anatomy papers of interest 

| Proteomics Percentile | Metabolomics Percentile | Title | Reference |
|----------|----------|----------|----------|
| 67 | 2 | Proteomics Standards Initiative: Fifteen Years of Progress and Future Work | [@doi:10.1021/acs.jproteome.7b00370] |
| 99 | 0 | Limited Environmental Serine and Glycine Confer Brain Metastasis Sensitivity to PHGDH Inhibition | [@doi:10.1158/2159-8290.cd-19-1228] |
| 100 | 0 | A high-throughput processing service for retention time alignment of complex proteomics and metabolomics LC-MS data| [@doi:10.1093/bioinformatics/btr094 ] |
| 0 | 100 | MeltDB: a software platform for the analysis and integration of metabolomics experiment data| [@doi:10.1093/bioinformatics/btn452] |
| 0 | 98 | In silico fragmentation for computer assisted identification of metabolite mass spectra| [@doi:10.1186/1471-2105-11-148] |
| 0 | 100 | The Metabonomic Signature of Celiac Disease| [@doi:10.1021/pr800548z] |
| 91 | 70 | Visualization of omics data for systems biology| [@doi:10.1186/1471-2105-11-148] |
| 0 | 16 | FunRich: An open access standalone functional enrichment and interaction network analysis tool| [@doi:10.1002/pmic.201400515] |
| 0 | 5 | Proteomic and Metabolomic Characterization of COVID-19 Patient Sera| [@doi:10.1016/j.cell.2020.05.032] |

Table 3: Proteomics/metabolomics papers of interest 

| Computational Biology Percentile | Human Genetics Percentile | Title | Reference |
|----------|----------|----------|----------|
| 99 | 0 | Development of Human Protein Reference Database as an Initial Platform for Approaching Systems Biology in Humans| [@doi:10.1101/gr.1680803] |
| 100 | 1 | A database for post-genome analysis| [@doi:10/cfgb98] |
| 100 | 1 | Use of mass spectrometry-derived data to annotate nucleotide and protein sequence databases| [@doi:10/ch565r] |
| 12 | 100| Genetic Discrimination: Perspectives of Consumers | [@doi:10.1126/science.274.5287.621] |
| 0 | 81 | Committee Opinion No. 690: Carrier Screening in the Age of Genomic Medicine | [@doi:10.1097/aog.0000000000001951 ] |
| 23 | 100 | Public health genomics: The end of the beginning| [@doi:10.1097/gim.0b013e31821024ca] |
| 100 | 99 | Initial sequencing and analysis of the human genome| [@doi:10.1038/35057062] |
| 100 | 100 | An STS-Based Map of the Human Genome| [@doi:10.1126/science.270.5244.1945] |
| 100 | 100 | A New Five-Year Plan for the U.S. Human Genome Project| [@doi:10.1126/science.8211127] |

Table 4: Computational biology/human genetics papers of interest 

![
The difference between percentile scores for four field pairs. 
](./images/combined_difference.png "Differences between fields' percentile scores"){#fig:percentiles width="100%"}

It is not possible to describe all the field-pairs and relevant differences between fields within the space of a journal article.
Instead, we have developed a web server that displays the percentiles for all pairs of fields in our dataset with at least 1000 shared articles.
The web server can be found at TODO.
