## Results

### Field-specific Citation Networks and Importances

We built citation networks using MeSH headings to denote fields.
We created a combined network and 100 shuffled versions for each pair of fields using a degree-preserving shuffling approach (see Methods).
We then split the original and shuffled networks to maintain the same set of papers within each constituent field, calculating the PageRank for all networks.
This process allows us to compare the manuscript's observed PageRank in each field with that constructed from a null that assumes no meaningful differences in citation patterns between fields.
Finally, we calculated percentile scores by comparing the PageRanks for the original and shuffled networks and made our results available at (TODO web server URL).

![ 
Schematic of the network analysis workflow. 
We construct networks of citations from pairs of MeSH headings, create shuffled versions of the graphs, split the networks into their component fields, and calculate the PageRank for each article in the networks.
](./images/schematic.png "Workflow schematic"){#fig:workflow width="100%"}

### Journal Importances Differ Between Fields

In an attempt to quantify the relative importance of journals, scientists have created rankings using metrics the Journal Impact Factor, which is essentially based on citations per article, and those that rely on more complex representations like Eigenfactor [@doi:10.5860/crln.68.5.7804]. 
It has previously been reported that journal rankings differ substantially between fields using metrics based on citation numbers [@doi:10.1007/s11192-022-04402-w].
We calculated a PageRank-based score for the journal as the median PageRank of manuscripts published in that journal for that field.
We first sought to understand the extent to which journal importance differences replicated using PageRank.

The journal *Science* provides an example of field-specific PageRank scores.
In the fields of nanotechnology or microscopy, the journal *Science* is the most influential journal based on median PageRank (Fig. @fig:journals A). <!--just to be clear, is this in each of these fields individually or in the field pair? if the field pair, how should that be interpreted?-->
However, for immunochemistry and anatomy, *Science* ranks fifth behind journals including *Nature* and *Cell* (Fig. @fig:journals B).
<!--are there other examples of journals with big differences?-->

<!--I'm not sure what to do with the below - is your argument that even specialty journals are often multi-disciplinary and consequently difficult to use as a proxy for importance? if so, what fraction of journals have 90% percent of their papers in only one "field" (MeSH heading here) vs what fraction have >1 MeSH heading with reasonable representation? It feels like something is missing in these sentences.-->
Further, while journals are sometimes used as a proxy measure for field [@doi:10.1007/s11192-020-03406-8], we find they often play host to interdisciplinary papers that may be more accurately assigned to a different field.
For example, "The metabonomic signature of celiac disease" is a paper with an extremely high PageRank in the field of metabolomics, but it was actually published in the *Journal of Proteome Research* [@doi:10.1021/pr800548z].

![
Differences between journal ranks across fields. Each point in the figure represents the median pagerank of all papers in the journal for the given fields.
](./images/combined_journals.png "Differences between journals ranks"){#fig:journals width="100%"}

### Manuscript PageRanks Differ Between Fields

We examined the univariate distributions of PageRank across fields (Fig. TODO).
To control for differences in the papers present, we also performed the analyses for field pairs using only the intersecting papers (Fig. TODO).
We found that fields differed in their PageRank distributions.
We also examined the correspondence of individual papers within the field-pair intersections.
We selected a representative field pair (Fig. @fig:heatmaps A), a field pair above the median (Fig. @fig:heatmaps B), and two high correlation field pairs (Fig. @fig:heatmaps C,D).
We found that many papers exhibited some correspondence between fields.
However, given the differences in univariate distributions, it remained difficult to determine which differences might be meaningful.

![
Heatmap of shared papers across fields
](./images/combined_heatmap.png "Heatmaps"){#fig:heatmaps width="100%"}

We devised a strategy to generate an empirical null for a field pair under the assumption that the field pair represented a single, homogenous field.
For each field-pair intersection, we performed a degree-distribution preserving permutation.
We created 100 permuted networks for each field pair.
We then calculated a percentile using the number of permuted networks with a lower PageRank for a manuscript than the true PageRank.
A manuscript with a PageRank higher than all networks has a percentile of 100, and one lower than all permuted networks has a percentile of zero. <!--Can we use percentiles here? Since you are generating 100 permutations it may be more natural. Will require multiplication by 100 for all the places you present it.-->
<!--at this point I am very confused about how what you are describing here is different than the first subsection-->
We used the difference in the percentile in each field as the field-specific affinity for a given paper.

<!--I feel like from here through the end of this section it just becomes a verbal description of the figures. Can you revise this to make it more clear how this relates to the premise of your paper (field-specific importances don't just apply to journals, they apply to manuscripts too?). It'd probably be good to have a lot less description of the figures. You may need to add some global comparisons/tables/etc - perhaps for the most extreme differences across the dataset and/or which fields have overlapping papers but the most differences vs. many overlapping papers and high similarities?-->
The nanotechnology/microscopy plot has a normal amount of variance, and clearly demonstrates the differences between fields (Fig. @fig:percentiles A).
In the center of the plot are papers that have similar percentile scores in both fields.
Towards the right side most papers have a large positive nanotech-microscopy score, indicating that they are more highly valued in the field of nanotechnology than in microscopy.
Such papers include "A robust DNA mechanical device controlled by hybridization topology" and "Bioadhesive poly(methyl methacrylate) microdevices for controlled drug delivery", while other papers such as "Turning the spotlight on cellular imaging" appropriately have a large negative nanotechnology-microscopy score despite falling far to the nanotech side of the distribution [@doi:10.1038/415062a; @doi:10/c7fpg4; @doi:10.1038/nbt1101-1013].
Meanwhile, the left side holds papers with strong negative scores, indicating the field of microscopy values the papers more.
The left side has papers like "Photostable luminescent nanoparticles as biological label for cell recognition of system lupus erythematosus patients" and "WSXM: a software for scanning probe microscopy and a tool for nanotechnology" [@doi:10.1166/jnn.2002.105; @horcas2007wsxm].
The papers with a high PageRank and similar percentile scores between fields are likely to be influential in both fields. 
In nanotechnology/microscopy these papers are frequently nanoscopic imaging papers like "Imaging intracellular fluorescent proteins at nanometer resolution", "In vivo imaging of quantum dots encapsulated in phospholipid micelles", and "Water-Soluble Quantum Dots for Multiphoton Fluorescence Imaging in Vivo" [@doi:10.1126/science.1127344; @doi:10.1126/science.1077194; @doi:10.1126/science.1083780].

The immunochemistry/anatomy fields have much higher correlation in their shared papers, but still have some outliers (Fig. @fig:percentiles B).
Examples of papers preferred in the immunochemistry field are "Immunoelectron microscopic exploration of the Golgi complex", "Immunocytochemical and electrophoretic analyses of changes in myosin gene expression in cat posterior temporalis muscle during postnatal development", and "Electron microscopic demonstration of calcitonin in human medullary carcinoma of thyroid by the immuno gold staining method" [@doi:10.1177/31.8.6863900; @doi:10.1007/bf01682147; @doi:10.1007/bf00514331].
Papers preferred in anatomy include "Grafting genetically modified cells into the rat brain: characteristics of E. coli Î²-galactosidase as a reporter gene", "Vitamin-D-dependent calcium-binding-protein and parvalbumin occur in bones and teeth", and "Mapping of brain areas containing RNA homologous to cDNAs encoding the alpha and beta subunits of the rat GABAA gamma-aminobutyrate receptor" [@doi:10/dptnbm; @doi:10.1007/bf02405306; @doi:10.1073/pnas.85.20.7815]
Papers with high PageRank in both domains and a similar importance are "Studies of the HER-2/neu Proto-Oncogene in Human Breast and Ovarian Cancer", "Expression of c-fos Protein in Brain: Metabolic Mapping at the Cellular Level", and "Proliferating cell nuclear antigen (PCNA) immunolocalization in paraffin sections: An index of cell proliferation with evidence of deregulated expression in some, neoplasms" [@doi:10.1126/science.2470152; @doi:10.1126/science.3131879; @doi:10.1002/path.1711620403].

Where immunochemistry/anatomy served as an example of a highly correlated pair of fields, papers in proteomics/metabolomics have very different PageRanks between fields (Fig. @fig:percentiles C). 
These fields illustrate one of the reasons why the difference in percentile scores is a useful metric: there is so much dispersion that it's hard to tell where a trend line should be.
When coloring by the difference in scores, however, it is easy to see which papers are valued more in each field.
The proteomics-heavy papers include "Proteomics Standards Initiative: Fifteen Years of Progress and Future Work", "Limited Environmental Serine and Glycine Confer Brain Metastasis Sensitivity to PHGDH Inhibition", and "A high-throughput processing service for retention time alignment of complex proteomics and metabolomics LC-MS data", [@doi:10.1021/acs.jproteome.7b00370;@doi:10/ghf85j;@doi:10.1093/bioinformatics/btr094] while the metabolomics papers have titles such as "MeltDB: a software platform for the analysis and integration of metabolomics experiment data", "In silico fragmentation for computer assisted identification of metabolite mass spectra", and "The Metabonomic Signature of Celiac Disease" [@doi:10.1093/bioinformatics/btn452;@doi:10/d7gpf5;@doi:10.1021/pr800548z].
The articles with high PageRank and importance in both fields include "Visualization of omics data for systems biology", "FunRich: An open access standalone functional enrichment and interaction network analysis tool", and "Proteomic and Metabolomic Characterization of COVID-19 Patient Sera" [@doi:10.1038/nmeth.1436;@doi:10.1002/pmic.201400515;@doi:10.1016/j.cell.2020.05.032].

The final pair, computational biology/human genetics, is interesting because while there is a good degree of variance in the PageRank values across fields, the highest rated papers in both fields had very similar percentile scores (Fig. @fig:percentiles D).
The high percentile difference papers in computational biology include several database papers [@doi:10.1101/gr.1680803; @doi:10/cfgb98; @doi:10/ch565r].
Their counterparts in human genetics were more on the policy paper side, likely because the human genetics MeSH subheading is distinct from the several other genetics subheadings including genetic research, population genetics, genomics, etc. [@doi:10.1126/science.274.5287.621; @doi:10.1097/aog.0000000000001951; @doi:10.1097/gim.0b013e31821024ca].
Meanwhile, the papers with high PageRanks in both fields tend to have similar percentile scores and be related to the genome sequencing and mapping [@doi:10.1038/35057062; @doi:10.1126/science.270.5244.1945; @doi:10/fwkrnb].

![
The difference between percentile scores for four field pairs. 
](./images/combined_difference.png "Differences between fields' percentile scores"){#fig:percentiles width="100%"}

### Here's a web app 

<!-- if this is going to be a section, something needs to be revised above - at the moment it's way short and just restates what's up there.-->
To demonstrate our findings, we have also set up a web server with all pairs of networks instead of solely the ones shown in this paper.
The web server can be found at X
